src/fabric/face.rs

```diff
@@ -45,4 +45,14 @@ impl Face {
             .map(|id| fabric.interval(id))
             .map(|Interval { omega_index, .. }| *omega_index)
     }
+
+    pub fn space(&self, fabric: &Fabric) -> Matrix4<f32> {
```

Comment:
Can you name this something more informative? Just `vector_space()` or something would be clearer